src/Workspaces/Core/Portable/Formatting/Rules/Operations/FormattingOperations.cs

```diff
@@ -5,6 +5,8 @@
 using Microsoft.CodeAnalysis.Shared.Utilities;
 using Microsoft.CodeAnalysis.Text;
 
+using static Microsoft.CodeAnalysis.Formatting.Rules.AdjustSpacesOption;
```

Comment:
💡  I would prefer to not have this. It makes the code below look like it's referencing constants in the current class.

Reply:
Changed.